translations/monero-core_da.ts

```diff
@@ -0,0 +1,2148 @@
+        <source>Transaction cost</source>
+        <translation>Transaktions pris</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="334"/>
+        <source>Payment ID &lt;font size=&apos;2&apos;&gt;( Optional )&lt;/font&gt;</source>
+        <translation>Betalings ID &lt;font size=&apos;2&apos;&gt;( Valgfri )&lt;/font&gt;</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="206"/>
+        <source>Slow (x0.25 fee)</source>
+        <translation>Langsom (x0,25 gebyr)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="207"/>
+        <source>Default (x1 fee)</source>
+        <translation>Standard (x1 gebyr)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="208"/>
+        <source>Fast (x5 fee)</source>
+        <translation>Hurtig (x5 gebyr)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="209"/>
+        <source>Fastest (x41.5 fee)</source>
+        <translation>Hurtigste (x41,5 fee)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="346"/>
+        <source>16 or 64 hexadecimal characters</source>
+        <translation>16 eller 64 hexadecimale tegn</translation>
+    </message>
+</context>
+<context>
+    <name>TxKey</name>
+    <message>
+        <location filename="../pages/TxKey.qml" line="90"/>
+        <source>Verify that a third party made a payment by supplying:</source>
+        <translation>Verficer at en tredjepart lavede en betaling ved at supplere:</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="95"/>
+        <source> - the recipient address</source>
+        <translation> - modtagers adresse</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="100"/>
+        <source> - the transaction ID</source>
+        <translation> - transaktions ID'et</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="105"/>
+        <source> - the secret transaction key supplied by the sender</source>
+        <translation> - den hemmelige transaktions nøgle suppleret af senderen</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="110"/>
+        <source>If a payment had several transactions then each must be checked and the results combined.</source>
+        <translation>hvis en betaling havde flere transaktioner så må hver eneste blive checket og resultaterne lagt sammen</translation>
+    </message>
+    <message>
+        <location filename="../pages/TxKey.qml" line="123"/>
+        <source>Address</source>
+        <translation>Adresse</translation>
+    </message>
+    <message>
+        <location filenam
```

Comment:
Missing `.` at the end

Reply:
ergg confused. 6 of those are for lines 1533-1536 with different errors